src/library/scala/collection/Map.scala

```diff
@@ -193,13 +193,15 @@ trait MapOps[K, +V, +CC[_, _] <: IterableOps[_, AnyConstr, _], +C]
     *  @return an immutable map consisting only of those key value pairs of this map where the key satisfies
     *          the predicate `p`. The resulting map wraps the original map without copying any elements.
     */
+  @deprecated("Use view.filterKeys. A future version will include a strict version of this method.", "2.13.0")
```

Comment:
I would say:

> Use `.view.filterKeys(p)`.

(The second sentence looks good)

We could maybe also document how to perform this operation in a strict way (`.view.filterKeys(p).toMap`)?

Reply:
@julienrf I'm on it